samples/server/petstore/rust-axum/output/ops-v3/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.13.0-SNAPSHOT
+7.12.0-SNAPSHOT
```

Comment:
can you please merge the latest master (upstream official) to your branch, build the CLI JAR locally and regenerate the samples again?